install/kubernetes/helm/istio/charts/galley/values.yaml

```diff
@@ -6,3 +6,23 @@ replicaCount: 1
 image: galley
 nodeSelector: {}
 
+# Specify the pod anti-affinity that allows you to constrain which nodes
+# your pod is eligible to be scheduled based on labels on pods that are
+# already running on the node rather than based on labels on nodes.
+# There are currently two types of anti-affinity:
+#    "requiredDuringSchedulingIgnoredDuringExecution"
+#    "preferredDuringSchedulingIgnoredDuringExecution"
+# which denote “hard” vs. “soft” requirements, you can define your values
+# in "podAntiAffinityLabelSelector" and "podAntiAffinityTermLabelSelector"
+# correspondingly.
+# For example:
+# podAntiAffinityLabelSelector:
+# - key: security
+#   operator: In
+#   value: S1
```

Comment:
would be good to give examples of multiple values.

Reply:
Yes, multiple `values` is also support, separated by comma, here is my local test file: https://github.com/istio/istio/pull/12691/commits/95748adce258b7abc39172357a26b7ef6ef7e0ef#diff-1aa45219f5b1d3d1013e6a11ce46ce37